test/test_fx.py

```diff
@@ -3701,9 +3701,9 @@ def tearDown(self):
         "leaky_relu": CONTROL_FLOW,
         "local_response_norm": CONTROL_FLOW,
         "margin_ranking_loss": CONTROL_FLOW,
-        "max_pool1d_with_indices": CONTROL_FLOW,
-        "max_pool2d_with_indices": CONTROL_FLOW,
-        "max_pool3d_with_indices": CONTROL_FLOW,
+        "max_pool1d_with_indices": ARG_TYPE_MISMATCH,
+        "max_pool2d_with_indices": ARG_TYPE_MISMATCH,
+        "max_pool3d_with_indices": ARG_TYPE_MISMATCH,
```

Comment:
This is the only part I don't understand (not saying it's right or wrong, I just don't understand FX)

Reply:
These tests are a bit strange: symbolic tracing in FX is expected to fail for these operations, and the tests are just verifying the error message of the first problem encountered during tracing. If code changes, the first error tracing runs into may change, causing these tests to fail. The change here LGTM as long as the tests pass.